provider/pipeline_evaluate_variables_stage.go

```diff
@@ -14,6 +14,7 @@ type evaluateVariablesStage struct {
 func newEvaluateVariablesStage() *evaluateVariablesStage {
 	return &evaluateVariablesStage{
 		baseStage: *newBaseStage(),
+		Variables: make(map[string]string),
```

Comment:
apply camelCase on `Variables`

Reply:
Variable was already defined in the code, I just initialized it.